bootstrap/bootstrap.go

```diff
@@ -164,22 +163,24 @@ func (g *Generator) Run() error {
 		defer os.Remove(f.Name()) // will not remove after rename
 	}
 
+	//nolint:gosec
```

Comment:
could we plz also write additional comment about why we are introducing `//nolint` directive?

Reply:
Added explanatory comments to all `//nolint` directives.